docs/user-guide/src/version_support.md

```diff
@@ -98,20 +101,22 @@ The table describes which branches/image-tags are tested in each periodic CI tes
-| metal3-periodic-ubuntu/centos-e2e-feature-test-main            | main         | main        | main           | latest         | latest      | latest     |
-| metal3-periodic-ubuntu/centos-e2e-feature-test-release-1-7     | release-1.7  | release-1.7 | release-0.6    | v0.6.1         | latest      | v24.1.1    |
-| metal3-periodic-ubuntu/centos-e2e-feature-test-release-1-6     | release-1.6  | release-1.6 | release-0.5    | v0.5.0         | latest      | v24.0.0    |
-| metal3-periodic-ubuntu/centos-e2e-feature-test-release-1-5     | release-1.5  | release-1.5 | release-0.4    | v0.4.0         | latest      | v23.1.0    |
+| INTEGRATION TESTS                                     | CAPM3 branch | IPAM branch | BMO branch/tag | Keepalived tag | MariaDB tag | Ironic tag |
+| ----------------------------------------------------- | ------------ | ----------- | -------------- | -------------- | ----------- | ---------- |
+| metal3-ubuntu/centos-e2e-integration-test-main        | main         | main        | main           | latest         | latest      | latest     |
+| metal3_main_integration_test_ubuntu/centos            | main         | main        | main           | latest         | latest      | latest     |
+| metal3-ubuntu/centos-e2e-integration-test-release-1-8 | release-1.8  | release-1.8 | release-0.8    | v0.8.0         | latest      | v25.0.0    |
+| metal3-ubuntu/centos-e2e-integration-test-release-1-7 | release-1.7  | release-1.7 | release-0.6    | v0.6.1         | latest      | v24.1.1    |
+| metal3-ubuntu/centos-e2e-integration-test-release-1-6 | release-1.6  | release-1.6 | release-0.5    | v0.5.0         | latest      | v24.0.0    |
+| metal3-ubuntu/centos-e2e-integration-test-release-1-5 | release-1.5  | release-1.5 | release-0.4    | v0.5.0         | latest      | v23.1.0    |
+
+| FEATURE AND E2E TESTS                                 | CAPM3 branch | IPAM branch | BMO branch/tag | Keepalived tag | MariaDB tag | Ironic tag |
+| ----------------------------------------------------- | ------------ | ----------- | -------------- | -------------- | ----------- | ---------- |
+| metal3-ubuntu/centos-e2e-feature-test-main            | main         | main        | main           | latest         | latest      | latest     |
+| metal3-ubuntu/centos-e2e-feature-test-release-1-8     | release-1.8  | release-1.8 | release-0.8    | v0.8.0         | latest      | v25.0.0    |
+| metal3-ubuntu/centos-e2e-feature-test-release-1-7     | release-1.7  | release-1.7 | release-0.6    | v0.6.1         | latest      | v24.1.1    |
+| metal3-ubuntu/centos-e2e-feature-test-release-1-6     | release-1.6  | release-1.6 | release-0.5    | v0.5.0         | latest      | v24.0.0    |
+| metal3-ubuntu/centos-e2e-feature-test-release-1-5     | release-1.5  | release-1.5 | release-0.4    | v0.4.0         | latest      | v23.1.0    |
```

Comment:
Same here, please add periodic. 
Also these are not correct trigger as those are divided in three parts. pivoting, remediation and features. we don't run any ubuntu feature test in CI periodics, only centos.
```
metal3-periodic-centos-e2e-feature-test-main-pivoting
metal3-periodic-centos-e2e-feature-test-main-remediation
metal3-periodic-centos-e2e-feature-test-main-features 
```
jenkins: https://jenkins.nordix.org/view/Metal3%20Periodic/